### PR TITLE
fix: content update

### LINKED ISSUE #127: 🐛 [BUG] - Update of the content in the cms is getting failed
### Description

When updating through toggle button or content dialog the status is not getting updated.
[problem.webm](https://github.com/user-attachments/assets/bd166f79-0f19-4713-8a0d-6f21cd240f5e)

### Reproduction steps

```bash
1. Go to 'Manage Content/CMS/Content Type'
2. Click on 'toggle button for the status in the listing page or inside modal'
3. Change the state.
4. See error
```

### Logs

_No response_

### Browsers

_No response_

### OS

_No response_

### PR DESCRIPTION
**Pull request by yassinedorbozgithub**
# Motivation
The main motivation is to fix the update content (CMS).

Fixes #127

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (Storybook)
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: api/src/cms/dto/content.dto.ts [hunk 1]
```diff
@@ -32,3 +32,19 @@ export class ContentCreateDto {
   @IsOptional()
   dynamicFields?: Record<string, any>;
 }
+
+export class ContentUpdateDto {
```
Comment (marrouchi) [inline api/src/cms/dto/content.dto.ts:36]: How about : 
```suggestion
export class ContentUpdateDto extends PartialType(ContentCreateDto) {}
```

File: frontend/src/components/contents/index.tsx [hunk 2]
```diff
@@ -165,8 +165,9 @@ export const Contents = () => {
                     field: "entity",
                     headerName: t("label.entity"),
                     flex: 1,
-                    valueGetter: (row: IContent) => {
-                      const contentType = getEntityFromCache(row.id);
+                    renderCell: ({ value }) => {
+                      const id = typeof value === "string" ? value : value.id;
```
Comment (marrouchi) [inline frontend/src/components/contents/index.tsx:169]: Do we still need this? is there a case where value s not a string ?

**Review by marrouchi — CHANGES_REQUESTED** [review]
Perfect ! 👏  Just a couple of comments

File: frontend/src/components/contents/index.tsx (hunk 2 above)
Comment (yassinedorbozgithub) [reply frontend/src/components/contents/index.tsx:169]: You are right, this update is no more needed

**Comment by yassinedorbozgithub** [comment]
> Perfect ! 👏 Just a couple of comments

Hello @marrouchi, all the required updates are pushed.
The PR is ready for the review process.

**Review by marrouchi — APPROVED** [review]
👏  , thanks also to @devamitranjan who had put some effort into this.

**Pull request merged**